desktop/TransactionManager.ts

```diff
@@ -409,7 +395,7 @@ class TransactionManager {
       layerReward: reward.layerReward.value.toNumber(),
       // layerComputed: reward.layerComputed.number, // TODO
       coinbase: `0x${coinbase}`,
-      smesher: toHexString(reward.smesher.id),
+      smesher: reward.smesher?.id ? toHexString(reward.smesher.id) : '0x00',
```

Comment:
is it possible to place the 0x00 to constant and define a name for it? not clear is it the default smasher from coinbase, or its incognito